airflow/security/permissions.py

```diff
@@ -32,6 +32,7 @@
 RESOURCE_JOB = "Jobs"
 RESOURCE_POOL = "Pools"
 RESOURCE_PLUGIN = "Plugins"
+RESOURCE_PROVIDERS = "Providers"
```

Comment:
```suggestion
RESOURCE_PROVIDER = "Providers"
```
Just for consistency (I know some of the naming conventions are annoying. It's a work in progress :)).